css/gallery.css

```diff
@@ -1,107 +1,130 @@
 /*class for the section of gallery*/
+.gallery-item::before {
+  content: " ";
+  display: block;
+  padding-top: 100%;
+}
 .gallery {
-  width: 635px;
-  height: 88px;
   font-family: Roboto;
   font-size: 16px;
   color: black;
-  padding: 60px 0;
+  padding-right: 20px;
 }
 .gallery__info {
-  top: 0;
-  left: 0;
-  margin-right: 150px;
-  margin-left: 150px;
-  margin-bottom: 17px;
-}
-.gallery__text {
-  padding-bottom: 17px;
-  margin-block-start: 0px;
-  margin-block-end: 0px;
+  display: flex;
+  flex-flow: row wrap;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-bottom: 30px;
```

Comment:
you could replace ,bottom,left and right with `margin: '0 20px 30px`, this meaning top = 0, sides = 20 and bottom = 30